crates/hdk/src/lib.rs

```diff
@@ -50,20 +50,20 @@
 //! HDK implements several key features:
 //!
 //! - Base HDKT trait for standardisation, mocking, unit testing support: [`hdk`] module
-//! - Capabilities and function level access control: [`capability`](crate::capability) module
```

Comment:
These are all redundancy warnings, no need to reference a specific module when it's already imported